operations/resourceblocks.md

```diff
@@ -71,7 +71,7 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
         {
             "AssignedResourceId": "5ee074b1-6c86-48e8-915f-c7aa4702086f",
             "CreatedUtc": "2016-03-29T22:02:34Z",
-            "EndUtc": "2016-01-01T16:00:00Z",
+            "EndUtc": "2016-01-01T16:00:00Z",            
             "Id": "5ab9d519-2485-4d77-85c4-2a619cbdc4e7",
             "IsActive": true,
             "StartUtc": "2016-01-01T10:00:00Z",
```

Comment:
issue: Please add `"DeletedUtc": null` to the example of active resourceblock under UpdatedUtc property (I can't comment on the line directly)